python/python_frontend/python_bindings.cpp

```diff
@@ -3720,7 +3722,14 @@ void initNvFuserPythonBindings(PyObject* module) {
          Tensor offsets,
          Tensor scale1,
          Tensor scale2,
-         std::optional<PrimDataType> dtype) -> Tensor {
+         std::optional<Tensor> alpha,
+         std::optional<Tensor> bias,
+         std::optional<Tensor> beta,
+         PrimDataType dtype,
+         int64_t output_block_scale_size,
+         PrimDataType output_block_scale_dtype,
+         bool output_gamma)
```

Comment:
Is gamma needed? If not, can we YAGNI and remove it? If yes, can we always output gamma? 

(I suspect it's needed for fp4 but not fp8.)